app/models/label/LabelTable.scala

```diff
@@ -84,16 +88,29 @@ object LabelTable {
   val auditTasks = TableQuery[AuditTaskTable]
   val gsvData = TableQuery[GSVDataTable]
   val labelTypes = TableQuery[LabelTypeTable]
+  val labelTags = TableQuery[LabelTagTable]
+  val tagTable = TableQuery[TagTable]
   val labelPoints = TableQuery[LabelPointTable]
   val labelValidations = TableQuery[LabelValidationTable]
   val missions = TableQuery[MissionTable]
+  val regions = TableQuery[RegionTable]
   val severities = TableQuery[LabelSeverityTable]
   val users = TableQuery[UserTable]
   val userRoles = TableQuery[UserRoleTable]
   val roleTable = TableQuery[RoleTable]
+  val descriptions = TableQuery[LabelDescriptionTable]
+  val temporariness = TableQuery[LabelTemporarinessTable]
 
   val labelsWithoutDeleted = labels.filter(_.deleted === false)
-  val labelsWithoutDeletedOrOnboarding = labelsWithoutDeleted.filter(_.tutorial === false)
+  val neighborhoods = regions.filter(_.deleted === false).filter(_.regionTypeId === 2)
+
+  // Grab city id of database and the associated tutorial street id for the city
+  val cityStr: String = Play.configuration.getString("city-id").get
+  val tutorialStreetId: Int = Play.configuration.getInt("city-params.tutorial-street-edge-id." + cityStr).get
+
+  // Filters out the labels placed during onboarding (aka panoramas that are used during onboarding
+  // Onboarding labels have to be filtered out before a user's labeling frequency is computed
+  val labelsWithoutDeletedOrOnboarding = labelsWithoutDeleted.filter(!_.tutorial)
```

Comment:
I think I prefer the `_.tutorial === false` over `!_.tutorial`. It just feels easier to read to me

Reply:
Ok